### PR TITLE
[Modules] DocumentDB - Add enable free tier tag to cosmos db properties

### PR DESCRIPTION
**Pull request by samuelya**
# Description

Added enablefreetier to properties of cosmos db to allow the deployment of cosmos db using free tier which will be useful for the development environment

Fixes: #3125 

> List any dependencies that are required for this change.

| Pipeline |
| - |
| [DocumentDB - DatabaseAccounts](https://github.com/samuelya/ResourceModules/actions/workflows/ms.documentdb.databaseaccounts.yml)|

# Type of Change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] Update to documentation

# Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (readme)
- [x] I did format my code

### CODE REVIEW

**Comment by samuelya** [comment]
@microsoft-github-policy-service agree

**Comment by ahmadabdalla** [comment]
@samuelya thanks for your contribution. Can you please attach a successful pipeline/workflow run for the change? Happy to support you if you need assistance in that.

**Comment by samuelya** [comment]
> @samuelya thanks for your contribution. Can you please attach a successful pipeline/workflow run for the change? Happy to support you if you need assistance in that.

Hi @ahmadabdalla Do I need to run the test on my own Azure subscription first using my own pipelines? Is that what you mean?

File: modules/Microsoft.DocumentDB/databaseAccounts/deploy.bicep [hunk 1]
```diff
@@ -35,6 +35,9 @@ param defaultConsistencyLevel string = 'Session'
 @description('Optional. Enable automatic failover for regions.')
 param automaticFailover bool = true
 
+@description('Optional. Enable free tier.')
```
Comment (ahmadabdalla) [inline modules/Microsoft.DocumentDB/databaseAccounts/deploy.bicep:38]:
```suggestion
@description('Optional. Flag to indicate whether Free Tier is enabled.')
```

**Review by ahmadabdalla — CHANGES_REQUESTED** [review]
Thanks @samuelya  :) And thanks for attaching the pipeline run. I've added a suggestion, can you please review. Also would require regenerating the documentation. Thanks

**Review by ahmadabdalla — APPROVED** [review]

**Pull request merged**